smoke-test/tests/cypress/cypress/e2e/searchV2/helpers/searchV2Helper.js

```diff
@@ -0,0 +1,54 @@
+import { hasOperationName } from "../../utils";
+import { SearchBarV2Helper } from "./searchBarV2Helper";
+
+export class SearchV2Helper {
+  searchBarV2 = new SearchBarV2Helper();
+
+  ensureItIsSeachPage() {
+    cy.url().should("include", "/search");
+  }
+
+  ensureUrlIncludesText(text) {
+    cy.url().should("include", text);
+  }
+
+  ensureIsOnEntityProfilePage(entityUrn) {
+    cy.url().should("include", `dataset/${entityUrn}`);
+  }
+
+  waitForSearchPageLoading() {
+    cy.wait(5000);
+  }
```

Comment:
same idea here - there should be something in the UI we could grab to let us know we're good to go that isn't an arbitrary wait

overall we're trying to move away from `wait` entirely if possible